sdk_container/src/third_party/coreos-overlay/profiles/coreos/base/package.accept_keywords

```diff
@@ -104,3 +104,7 @@
 
 # Accept unstable host Rust compilers.
 =virtual/rust-1.76.0 ~amd64 ~arm64
+
+# Upgrade to latest version for secureboot
+=sys-boot/mokutil-0.6.0 ~amd64
+>>>>>>> 8c26cea964 (coreo-base/coreos: Add mokutil to the base amd64 image)
```

Comment:
I think this line got missed during rebase.

Reply:
Thanks, updated